rosetta-source/src/main/rosetta/product-qualification-func.rosetta

```diff
@@ -1441,8 +1435,7 @@ func Qualify_ForeignExchange_VanillaOption: <"Qualifies a product as FX Plain Va
         Qualify_AssetClass_ForeignExchange(economicTerms) = True
             and economicTerms -> payout -> optionPayout exists
             and economicTerms -> payout -> optionPayout -> underlier -> foreignExchange only exists
-            and (economicTerms -> payout -> optionPayout -> exerciseTerms -> optionStyle -> americanExercise exists
-                or economicTerms -> payout -> optionPayout -> exerciseTerms -> optionStyle -> europeanExercise exists)
+            and (economicTerms -> payout -> optionPayout -> exerciseTerms -> style exists)
```

Comment:
This needs to test that `style` is either `Single` or `Continuous` (or alternatively isn't `Multiple`)